dynawo/sources/Models/Modelica/Dynawo/Electrical/Lines/dynamicLine.mo

```diff
@@ -0,0 +1,86 @@
+within Dynawo.Electrical.Lines;
+model dynamicLine "AC power line - PI model considerating the transistent mode "
+
+  import Dynawo.Types;
+  import Dynawo.Connectors;
+  import Dynawo.Electrical.Controls.Basics.SwitchOff;
+  extends SwitchOff.SwitchOffLine;
+  extends AdditionalIcons.Line;
+
+  Connectors.ACPower terminal1(V(re(start = u10Pu.re), im(start = u10Pu.im)), i(re(start = i10Pu.re), im(start = i10Pu.im))) annotation(
+    Placement(visible = true, transformation(origin = {-100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Connectors.ACPower terminal2(V(re(start = u20Pu.re), im(start = u20Pu.im)), i(re(start = i20Pu.re), im(start = i20Pu.im))) annotation(
+    Placement(visible = true, transformation(origin = {100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+
+
+  parameter Types.PerUnit RPu=0.00375 "Resistance in pu Current through the equivalent impedance G+jB on side 1 in pu (base UNom, SnRef) (base SnRef)";
+  parameter Types.PerUnit XPu=0.0375 "Reactance in pu (base SnRef)";
+  parameter Types.PerUnit GPu=0 "Half-conductance i=u20Pu*ComplexMath.conj(i20Pu)n pu (base SnRef)";
+  parameter Types.PerUnit BPu=0.0000375 "Half-susceptance in pu (base SnRef)";
+
+  parameter Types.PerUnit Iz0Pu=-11.1008 "Start value of the current through the equivalent impedance R+jX in pu (base UNom, SnRef)";
+  parameter Types.Angle Iz0Phase=0.0433 "Start value of the angle of the current through the equivalent impedance R+jX in rad";
```

Comment:
Could be named IRLPhase0 "Start value of the current angle in the R,L part of the line in rad"